packages/components/src/popover/index.js

```diff
@@ -163,26 +163,22 @@ const Popover = (
 		: placementProp;
 
 	const ownerDocument = useMemo( () => {
-		let documentToReturn;
-
 		if ( anchorRef?.top ) {
-			documentToReturn = anchorRef?.top.ownerDocument;
+			return anchorRef?.top.ownerDocument;
```

Comment:
This change is not needed here.. Leftover from the rebase.

Reply:
I think this change was never needed. It was just a code improvement Riad included.